airflow/ui/src/pages/Dashboard/HistoricalMetrics/AssetEvents.tsx

```diff
@@ -0,0 +1,111 @@
+/*!
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import {
+  Box,
+  Heading,
+  Flex,
+  HStack,
+  VStack,
+  StackSeparator,
+  Skeleton,
+} from "@chakra-ui/react";
+import { createListCollection } from "@chakra-ui/react/collection";
+
+import { useAssetServiceGetAssetEvents } from "openapi/queries";
+import { MetricsBadge } from "src/components/MetricsBadge";
+import { Select } from "src/components/ui";
+
+import { AssetEvent } from "./AssetEvent";
+
+type AssetEventProps = {
+  readonly assetSortBy: string;
+  readonly endDate: string;
+  readonly setAssetSortBy: React.Dispatch<React.SetStateAction<string>>;
+  readonly startDate: string;
+};
+
+export const AssetEvents = ({
+  assetSortBy,
+  endDate,
+  setAssetSortBy,
+  startDate,
+}: AssetEventProps) => {
+  const { data, isLoading } = useAssetServiceGetAssetEvents({
+    limit: 6,
+    orderBy: assetSortBy,
+    timestampGte: startDate,
+    timestampLte: endDate,
+  });
+
+  const assetSortOptions = createListCollection({
+    items: [
+      { label: "Newest first", value: "-timestamp" },
+      { label: "Oldest first", value: "timestamp" },
+    ],
+  });
+
+  return (
+    <Box borderRadius={5} borderWidth={1} ml={2} pb={2}>
+      <Flex justify="space-between" mr={1} mt={0} pl={3} pt={1}>
+        <HStack>
+          <MetricsBadge
+            backgroundColor="blue.solid"
+            runs={isLoading ? 0 : data?.total_entries}
+          />
+          <Heading marginEnd="auto" size="md">
+            Asset Events
+          </Heading>
+        </HStack>
+        <Select.Root
+          collection={assetSortOptions}
+          data-testid="asset-sort-duration"
+          defaultValue={["-timestamp"]}
+          onValueChange={(option) => setAssetSortBy(option.value[0] as string)}
+          variant="outline" // Flushed seems to be right option to remove border but is not valid as per typescript
```

Comment:
What does this comment mean?

Reply:
In the UI design the select component has no border in asset events widget. "flushed" is a variant that achieves this effect on select but the type for select only has outline | subtle as valid values though flushed works. This can be kept outline too and would just deviate from the design.